redis/src/main/scala/zio/redis/RespValue.scala

```diff
@@ -72,19 +72,25 @@ object RespValue {
       }
   }
 
-  private[redis] final val Decoder: Transducer[RedisError.ProtocolError, Byte, RespValue] = {
-    import internal.State
+  private[redis] final val Decoder: ZPipeline[Any, RedisError.ProtocolError, Byte, Option[RespValue]] = {
+    val Sinker: ZSink[Any, RedisError.ProtocolError, String, String, Option[RespValue]] = {
```

Comment:
Let's drop the type, and use the lower camel case.

Reply:
Please don't mark unresolved issues as resolved.

Reply:
Oh, sorry, I thought it was just the outermost type declaration that should be removed.